ament_copyright/ament_copyright/parser.py

```diff
@@ -54,24 +56,17 @@ def read(self):
     def parse(self):
         raise NotImplemented()
 
-    def identify_copyright(self):
-        known_copyrights = get_copyright_names()
-        for c in self.copyrights:
-            found_name = c.name
-            for identifier, name in known_copyrights.items():
-                if name == found_name:
-                    self.copyright_identifiers.append(identifier)
-                    break
-            else:
-                self.copyright_identifiers.append(UNKNOWN_IDENTIFIER)
-
     def identify_license(self, content, license_part):
+        if content is None:
+            return
+
         for name, license in get_licenses().items():
-            if content is not None and getattr(license, license_part) == content:
+            template = getattr(license, license_part).replace('\n', ' ')
+            tomatch = content.replace('\n', ' ')
+            x = difflib.SequenceMatcher(a=template, b=content)
+            if x.ratio() > 0.95:
```

Comment:
I guess, though a license that is 95% the same as this would probably be considered the same.

So the real question is whether we should just up this value (98%, say), or do you disagree with the approach?